rs/types/types/src/crypto/threshold_sig/ni_dkg/id.rs

```diff
@@ -66,8 +70,22 @@ impl TryFrom<NiDkgIdProto> for NiDkgId {
                 PrincipalId::try_from(ni_dkg_id_proto.dealer_subnet.as_slice())
                     .map_err(NiDkgIdFromProtoError::InvalidPrincipalId)?,
             ),
-            dkg_tag: NiDkgTag::try_from(ni_dkg_id_proto.dkg_tag)
-                .map_err(|_| NiDkgIdFromProtoError::InvalidDkgTag)?,
+            dkg_tag: {
+                match ni_dkg_id_proto.dkg_tag {
+                    1 => Ok(NiDkgTag::LowThreshold),
+                    2 => Ok(NiDkgTag::HighThreshold),
```

Comment:
Should we return an error if the tag is `LowThreshold` or `HighThreshold`, but `ni_dkg_id_proto.key_id` *isn't* `None`?

Reply:
+1

Reply:
Done: https://github.com/dfinity/ic/pull/2445/commits/b5eb7bd2fb76a523bbfe23dbe60261a5a9ae5172

Reply:
Resolving this comment, because it is addressed (see above). Let me know if you have further comments.